### DISCUSSION TITLE
Data drift and model monitoring features

### DISCUSSION THREAD

**Discussion by shubhamkansal1993** [Azure/mlops-v2#102] [General]
This template is very useful. Wanted to know if there are plans to add model and data drift monitoring pipelines for model retraining?

**Comment by setuc** [comment]
There are pipelines available that take care of monitoring and have data drift models built in. Take a look at the configuration file and set enable-monitoring to true. That will also deploy the solution for the monitoring and solution. We are working on a more integrated solution, and you can expect an update very soon.
